src/components/views/auth/PassphraseConfirmField.tsx

```diff
@@ -77,6 +77,7 @@ class PassphraseConfirmField extends PureComponent<IProps> {
                 value={this.props.value}
                 onChange={this.props.onChange}
                 onValidate={this.onValidate}
+                autoFocus={this.props.autoFocus}
```

Comment:
If we're defining `IProps` in terms of  `Omit<IInputProps, ...>` then it seems like we should rather strip out the `props` we know we *don't* want to pass through, and then pass through all the rest, otherwise this sort of thing will keep happening.

Maybe a job for another day though.

Reply:
Indeed, not sure why the props were defined like that without a splat pass-through but I'd argue a job for another day